### ISSUE TITLE
July 2017 Missing Alt Text

### ISSUE THREAD

**Issue by adamdoe** [marketingincolor/mad-sungard-website#1]
@eddt 

File Saved Here:

P:\Clients\Madico\Website & Blogs\SunGard_SEO

It looks like we'll need to add alt text into the theme. If you want me to look at this please let me know.
